project/docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine
+MAINTAINER Benedict Lau <benedict.lau@groundupworks.com>
+
+RUN apk add --update-cache
+RUN apk add autoconf automake build-base libtool ncurses nodejs nodejs-npm python
```

Comment:
This can definitely be improved to cut down 70% of image size. Now it's > 300 MB.